application/phoenix/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM elixir:1.9.4
+
+RUN curl -sLJO https://deb.secrethub.io/amd64
+RUN dpkg -i secrethub-cli-amd64.deb
+RUN rm secrethub-cli-amd64.deb
```

Comment:
Would be nice to switch to the Alpine image here if possible:

```suggestion
FROM elixir:1.9.4-alpine

# Install SecretHub
RUN apk add --repository https://alpine.secrethub.io/alpine/edge/main --allow-untrusted secrethub-cli
```

Reply:
Alright, I'll try this now!

Reply:
Works. just as fine, I'm changing it now. Thanks for the suggestion!